yarn.lock

```diff
@@ -21584,19 +21579,6 @@ react-color@2.18.0:
     reactcss "^1.2.0"
     tinycolor2 "^1.4.1"
 
-react-color@^2.17.0:
-  version "2.19.3"
-  resolved "https://registry.yarnpkg.com/react-color/-/react-color-2.19.3.tgz#ec6c6b4568312a3c6a18420ab0472e146aa5683d"
-  integrity sha512-LEeGE/ZzNLIsFWa1TMe8y5VYqr7bibneWmvJwm1pCn/eNmrabWDh659JSPn9BuaMpEfU83WTOJfnCcjDZwNQTA==
-  dependencies:
-    "@icons/material" "^0.2.4"
```

Comment:
Yeah, `@icons/material` didn't change and it's used by react-color.

I only manually changed the resolution for `react-color` like this:

```
react-color@2.18.0, react-color@^2.17.0:
...
```

my suspect is that since yarn was resolving 2 different versions of `react-color` it wasn't able to hoist it in the top-level `node_mosules`, but `@icons/material` resolves to `0.2.4` for both so it's being hoisted. since internally `react-color` does `var _UnfoldMoreHorizontalIcon = require('@icons/material/UnfoldMoreHorizontalIcon');` i suspect the node resolution algorithm (described in https://nodejs.org/api/modules.html) fails to find `UnfoldMoreHorizontalIcon` as `@icons/material` it's not in the same dir.

Reply:
Yeah that makes sense. Thanks the explanation @Elfo404. 👍